webapp/channels/src/components/sidebar_right_menu/sidebar_right_menu.tsx

```diff
@@ -26,52 +24,53 @@ type Props = {
-        GlobalActions.emitUserLoggedOutEvent();
-    };
-
-    render() {
-        let siteName = '';
-        if (this.props.siteName != null) {
-            siteName = this.props.siteName;
-        }
-        let teamDisplayName = siteName;
-        if (this.props.teamDisplayName) {
-            teamDisplayName = this.props.teamDisplayName;
-        }
+const SidebarRightMenu = ({
+    siteName: defaultSiteName,
+    teamDisplayName: defaultTeamDisplayName,
+    isOpen,
+    isMobileView,
+}: Props) => {
+    let siteName = '';
+    if (defaultSiteName != null) {
+        siteName = defaultSiteName;
+    }
+    let teamDisplayName = siteName;
+    if (defaultTeamDisplayName) {
+        teamDisplayName = defaultTeamDisplayName;
+    }
 
-        return (
-            <div
-                className={classNames('sidebar--menu', {'move--left': this.props.isOpen && this.props.isMobileView})}
-                id='sidebar-menu'
-            >
-                <div className='team__header theme'>
-                    <Link
-                        className='team__name'
-                        to={`/channels/${Constants.DEFAULT_CHANNEL}`}
-                    >
-                        {teamDisplayName}
-                    </Link>
-                </div>
+    return (
+        <div
+            className={classNames('sidebar--menu', {'move--left': isOpen && isMobileView})}
+            id='sidebar-menu'
+        >
+            <div className='team__header theme'>
+                <Link
+                    className='team__name'
+                    to={`/channels/${Constants.DEFAULT_CHANNEL}`}
+                >
+                    {teamDisplayName}
+                </Link>
+            </div>
 
-                <div className='nav-pills__container mobile-main-menu'>
-                    <CSSTransition
-                        in={this.props.isOpen && this.props.isMobileView}
-                        classNames='MobileRightSidebarMenu'
-                        enter={true}
-                        exit={true}
-                        mountOnEnter={true}
-                        unmountOnExit={true}
-                        timeout={{
-                            enter: ANIMATION_DURATION,
-                            exit: ANIMATION_DURATION,
-                        }}
-                    >
-                        <MainMenu mobile={true}/>
-                    </CSSTransition>
-                </div>
+            <div className='nav-pills__container mobile-main-menu'>
+                <CSSTransition
+                    in={isOpen && isMobileView}
+                    classNames='MobileRightSidebarMenu'
+                    enter={true}
+                    exit={true}
+                    mountOnEnter={true}
+                    unmountOnExit={true}
+                    timeout={{
+                        enter: ANIMATION_DURATION,
+                        exit: ANIMATION_DURATION,
+                    }}
```

Comment:
This is out of the scope, but since I am asking for changes...

Do you mind moving this to a constant outside of the component? The values are always the same, and this way we avoid passing a new object on every render (and even creating the object on every render).